content/posts/2018-03-18-dependency-upgrade.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "The Art of a Dependency Upgrade"
+description: "A walk to a technical marvel without breaking a leg or two."
```

Comment:
like you're a professional writer lol (it's good) 👍

Reply:
Ahhh, you make me shy 💋